metroae

```diff
@@ -642,13 +644,24 @@ function audit_log_and_exit {
     print_version_and_exit $1
 }
 
+function setup_log_files {
+    debug ${FUNCNAME[0]}
+    mkdir $LOGS_DIRECTORY
+    touch $ANSIBLE_LOG
+    touch $AUDIT_LOG
+    current_time=$(date "+%Y.%m.%d-%H.%M.%S")
+    ln -s $ANSIBLE_LOG ansible.log.$current_time
```

Comment:
you can combine lines 653 and 654: `ln -s $ANSIBLE_LOG ansible.log.$WORKFLOW.$DEPLOYMENT.$current_time`